### ISSUE TITLE
Please add support for bytestring-lexing-0.5.x

### ISSUE THREAD

**Issue by peti** [informatikr/hedis#25]
We would like to distribute hedis-0.6.5 in NixOS, but our attempts to build it currently fail because we've updated to bytestring-lexing-0.5.x already, which the build doesn't support.

**Comment by peti** [comment]
Will you release a new version?

**Comment by informatikr** [comment]
I did.

**Comment by peti** [comment]
Citing from http://hydra.cryp.to/build/925750/nixlog/2/raw:

~~~~~~~~~~~~~~
Building hedis-0.6.6...
Preprocessing library hedis-0.6.6...

src/Database/Redis/Types.hs:1:33: Warning:
    -XOverlappingInstances is deprecated: instead use per-instance pragmas OVERLAPPING/OVERLAPPABLE/OVERLAPS

src/Database/Redis/Types.hs:8:8:
    Could not find module ‘Data.ByteString.Lex.Double’
    Use -v to see a list of the files searched for.
~~~~~~~~~~~~~~~~~~

Did you test the new version before uploading?

**Comment by dmjio** [comment]
Getting the same as @peti 
```haskell
Preprocessing library hedis-0.6.6...

src/Database/Redis/Types.hs:1:33: Warning:
    -XOverlappingInstances is deprecated: instead use per-instance pragmas OVERLAPPING/OVERLAPPABLE/OVERLAPS

src/Database/Redis/Types.hs:8:8:
    Could not find module ‘Data.ByteString.Lex.Double’
    Use -v to see a list of the files searched for.
```

**Comment by informatikr** [comment]
Should be fixed in version 0.6.7 on Hackage.

Thanks for the reports.

**Comment by dmjio** [comment]
Thanks !
